[Feedback] [docrevamp] [devcommunity] (#2900)
MicrosoftDocs/msteams-docs: msteams-platform/feedback.md

> |  **Documentation issues**  |        Choose `This page` option in the **Submit and view feedback** for section, available at bottom of respective documentation to open an [issue](https://github.com/MicrosoftDocs/msteams-docs/issues) on the Microsoft Teams documentation GitHub repository.                                                                                                                                                                                            |

Reviewer: ```suggestion
|  **Documentation issues**  |        Select `This page` in the **Submit and view feedback** section available at bottom of the documentation to open an [issue](https://github.com/MicrosoftDocs/msteams-docs/issues) on the Microsoft Teams documentation GitHub repository.                                                                                                                                                                                            |
```

> |  **Documentation updates**           |     Select the **Edit** link in an article and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repository.                                                                                                                                                           |

Reviewer: ```suggestion
|  **Documentation updates**           |     Select **Edit** in the article you want to update and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repository.                                                                                                                                                           |
```

> |       **Features suggestions**       |                                                                                                                                                                      You can post features suggestions on (https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform.                                                                                                                                                                      |

Reviewer: ```suggestion
|       **Features suggestions**       |                                                                                                                                                                      Post features suggestions on (https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform.                                                                                                                                                                      |
```
```

---

# Microsoft Teams developer community channels

Microsoft Teams community of developers use Stack Overflow to connect with other developers to ideate, get clarifications, and submit queries. In addition, you can also use one of the following contacts or sites depending on the type of support required to submit your queries, doubts, or clarifications. You can also get product specific information, such as issues, failures, and general questions about Microsoft Teams: 

|            **Support type**            |               **Contact**                                                                                  |
|-----------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> |         **Community support**          |<ol><li> Use the `botframework & microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams). Stack Overflow has guidelines, such as requiring a descriptive title, a complete and concise problem statement, and sufficient details to reproduce your issue. Feature requests or overly broad questions are off-topic. For guidance on how to ask your queries, users must visit the [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask). </li>                                                                                                                                                                       <li>  Questions can also be posted at [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) using the `office-teams-app-dev` tag. We love Stack Overflow and our extensive support is assured for customers who ask questions there. However, Stack Overflow has specific criteria about what questions are appropriate for the community and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this.  </li> </ol>                                                                                                  |

Reviewer: Please rephrase as follows:

You can post questions on one of the following:

- [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams): Use the microsoft-teams tag to post the questions.
   > [!NOTE]
   > Ensure that you follow the Stack Overflow guidelines such as provide a descriptive title, a complete and concise problem statement, and sufficient detail to reproduce your issue. Feature requests or overly broad questions are off-topic. if you are a new user, for more information, see [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask).
   
- Rephrase in the similar way for point 2.

> |        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). Use the `microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams).                                                                                                                                                                                                                       |

Reviewer: Global: Please make the list in alphabetical order

> |        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). Use the `microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams).                                                                                                                                                                                                                       |

Reviewer: Change to:

Tab SDK issues or suggestions

> |        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). Use the `microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams).                                                                                                                                                                                                                       |

Reviewer: Change the tone to active

|            **Bot & Messaging Extension SDK issues/suggestions**             |      Submit issues and feature requests to the SDK repository for your bot's language ([C#](https://github.com/Microsoft/botbuilder-dotnet/), [Javascript](https://github.com/Microsoft/botbuilder-js), or [Python](https://github.com/Microsoft/botbuilder-python)). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework%20microsoft-teams) using the `botframework & microsoft-teams` tag.                                                                                            |
| **Samples** |             Submit issues with samples to respective [Microsoft Teams Samples](/microsoftteams/platform/tutorials/code-samples) repository.                                                                                                                                                                                            |

> | **Features suggestions**             |      Make your suggestions for a feature on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |

Reviewer: You can post features suggestions on (https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform.

## Product support channels
The following table lists the support channels to submit Teams product related queries:

|            **Support type**            |               **Contact**                                                                                  |
|-----------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|         **Microsoft Teams product issues and failures**          | Tenant Admin can reach out to [Microsoft 365 Product Support](/microsoft-365/admin/contact-support-for-business-products).                                                            |

> |        **General questions about Microsoft Teams**        |  Submit your questions on [Microsoft Teams Community](https://answers.microsoft.com/en-us/msteams/forum).               |                                                           

Reviewer: Please maintain parallelism across the table.

## Support type

Choose the correct support type to submit your issue and keep yourself updated with the Teams app development.

|            **Support type**            |               **Submit your issues or queries**                                                                                  |
|-----------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> |         **Community support**          |<ol><li> Questions can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams) using the `microsoft-teams` tag.  Stack Overflow has guidelines, such as requiring a descriptive title, a complete and concise problem statement, and sufficient details to reproduce your issue. Feature requests or overly broad questions are off-topic. New users must visit the [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask) for more details. </li>                                                                                                                                                                       <li>  Questions can also be posted at [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) using the `office-teams-app-dev` tag. We love Stack Overflow and our extensive support is assured for customers who ask questions there. However, Stack Overflow has specific criteria about what questions are appropriate for the community and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this.  </li> </ol>                                                                                                  |

Reviewer 2: ```suggestion
|         **Community support**          |<ol><li> Questions can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams) using the `microsoft-teams` tag.  Stack Overflow has guidelines, such as requiring a descriptive title, a complete and concise problem statement, and sufficient details to reproduce your issue. Feature requests or overly broad questions are off-topic. For guidance on how to ask your queries, users must visit the [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask). </li>                                                                                                                                                                       <li>  Questions can also be posted at [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) using the `office-teams-app-dev` tag. We love Stack Overflow and our extensive support is assured for customers who ask questions there. However, Stack Overflow has specific criteria about what questions are appropriate for the community and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this.  </li> </ol>                                                                                                  |
```

> |        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams) using the `microsoft-teams` tag.                                                                                                                                                                                                                       |

Reviewer 2: ```suggestion
|        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). Use the `microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams).                                                                                                                                                                                                                       |
```

> |            **Bot & Messaging Extension SDK issues/suggestions**             |      Submit issues and feature requests to the SDK repository for your bot's language ([C#](https://github.com/Microsoft/botbuilder-dotnet/), [Javascript](https://github.com/Microsoft/botbuilder-js), or [Python](https://github.com/Microsoft/botbuilder-python)). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework%20microsoft-teams) using the `botframework & microsoft-teams` tag.                                                                                            |

Reviewer 2: Use the `botframework & microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework%20microsoft-teams).

| **Samples** |             Submit issues with samples to respective [Microsoft Teams Samples](/microsoftteams/platform/tutorials/code-samples) repository.                                                                                                                                                                                            |

> | **Features suggestions**             |      Make your feature's suggestions on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |

Reviewer 2: Do features make suggestions?
```suggestion
| **Features suggestions**             |      Make your suggestions for a feature on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |
```

|        **Documentation issues**        |                                                                                                                                                                      Choose `This page` option in the **Submit and view feedback for** section, available at bottom of respective documentation to open an [issue](https://github.com/MicrosoftDocs/msteams-docs/issues) on the Microsoft Teams documentation GitHub repository.                                                                                                                                                                      |
|       **Documentation updates**        | Select the **Edit** link in an article and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repository.                                                                                                                                                                      |

> |       **Community chat group**        | Select **SIGN IN TO START TALKING** to join the informal community chat room for [Microsoft Teams app developers](https://gitter.im/OfficeDev/MicrosoftTeamsAppDev) if you are logging in for the first time. Select **JOIN ROOM**, if you are already logged in. |

Reviewer 2: check the punctuations.
replace "logging" with "signing"

> |          **General questions**         |          General questions about the Microsoft Teams developer platform must be sent to [Microsoft Teams Community Help](mailto:microsoftteamsdev@microsoft.com). We encourage posting questions on channels mentioned and use email only if none of the above are applicable.                                                                                                                                                                          |

Reviewer 2: ```suggestion
|          **General questions**         |          Send general questions about the Microsoft Teams developer platform to [Microsoft Teams Community Help](mailto:microsoftteamsdev@microsoft.com). We encourage posting questions on channels mentioned, and use email only if no other mode of communication is applicable.                                                                                                                                                                          |
```

## Product Support channels

> Support channels to submit your Teams product related questions:

Reviewer 2: The following table lists the support channels to submit Teams product related queries:

> |            **Support type**            |               **Submit your issues or queries**                                                                                  |

Reviewer 2: Why submit your issues or queries, as a table header? Are we not giving them where or whom to contact?
Suggest using Contact.
